7-gce/gce/deploy.sh

```diff
@@ -98,12 +100,14 @@ gcloud compute http-health-checks create ah-health-check \
 # [START create_backend_service]
 gcloud compute backend-services create $SERVICE \
   --http-health-checks ah-health-check
+  --global
```

Comment:
Param _--global_ is specified so that the user will not be asked to choose a region from 10+ choices when running the script.